src/zenml/cli/stack.py

```diff
@@ -1141,19 +1141,49 @@ def _import_stack_component(
 @stack.command("import", help="Import a stack from YAML.")
 @click.argument("stack_name", type=str, required=True)
 @click.argument("filename", type=str, required=False)
+@click.option(
+    "--yes",
+    "-y",
+    "force",
+    is_flag=True,
+    help="Force the import of stack components irrespective of ZenML version matching.",
+)
+@click.option(
```

Comment:
I think this entire option should be removed to be honest. We only added the deprecated options because we previously had the old option and didn't want it to break for users. But when implementing this functionality for some entirely new feature, there is no need for this.

Similar to my message above, I'm also not sure whether `--force` or `--yes` would be the right flag here. In this case it's not automatically confirming something, but instead "forcing" the import even if the versions don't match. Personally something like `--ignore-version-mismatch` would be my preferred option, but I think others might disagree and prefer the ease of use of just using the same flag everywhere.

Reply:
I totally agree with the `--ignore-version-mismatch` idea. I would prefer something that is explicit about what it does too.